service/src/main/java/tds/config/endpoints/ConfigController.java

```diff
@@ -0,0 +1,48 @@
+package tds.config.endpoints;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import tds.config.ClientSystemFlag;
+import tds.config.ClientTestProperty;
+import tds.config.services.ConfigService;
+import tds.config.web.resources.ClientSystemFlagResource;
+import tds.config.web.resources.ClientTestPropertyResource;
+
+
+@RestController
+@RequestMapping("/config")
+public class ConfigController {
+    private static final Logger LOG = LoggerFactory.getLogger(ConfigController.class);
+    private final ConfigService configService;
+
+    @Autowired
+    public ConfigController(ConfigService configService) {
+        this.configService = configService;
+    }
+
+    @RequestMapping(value = "/isAlive", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseBody
```

Comment:
Yep - the idea behind the `isAlive` method is to act as the load balancer's keep alive feature.  I like the idea of having some sort of base controller, but I'm a bit hesitant/gun-shy of a shared code project.  Right this second, I think copying/pasting the isAlive into other controllers is ghetto, but okay.